nautobot/core/templatetags/helpers.py

```diff
@@ -795,7 +795,13 @@ def saved_view_modal(
         "clear_view",
     ]
 
-    table_name = lookup.get_table_for_model(model).__name__
+    view_class = lookup.get_view_for_model(model, "List")
```

Comment:
Just confirming - this works for both legacy views and NautobotUIViewSet views?

Reply:
Yeah, get_view_for_model covers both. In the case of NautobotUIViewSet, the "List" parameter is unused.